### ISSUE TITLE
Can we have an option to disable ID3v2.2-3 separator?

### ISSUE THREAD

**Issue by gnattu** [Zeugma440/atldotnet#281]
## The problem

Currently the ID3v2.2/2.3 tagged artists will get automatically separated with the predefined separator in ATL (the `/` character). However, that character is also common among artists' names. For our use case, we allow the user to have a more customizable control on the splitting behavior: https://github.com/jellyfin/jellyfin/pull/12385, which makes the internal split of ATL a bit redundant.

## Environment

* ATL version (or git revision) that exhibits the issue: 6.04
* Last ATL version that did not exhibit the issue (if applicable): Yes
* OS/version used to run ATL: All dotnet 8

## Details

The behavior is likely implemented here:

https://github.com/Zeugma440/atldotnet/blob/0532913a2aa7ba7284fcc0693de9404c50bf32cc/ATL/AudioData/IO/ID3v2.cs#L1958-L1961

## Code To Reproduce Issue [ Good To Have ]

Having a file `sample.mp3` with Artist tag `A/BC` in ID3v2.3

```
Track track = new Track("sample.mp3");
var artist = track.Artist // `/` will be replaced by `;`
```

It would be good to have an option to disable this in `ATL.Settings` to disable splitting for ID3 versions below ID3v2.4

**Comment by gnattu** [comment]
Yes, we are using `ATL.Settings.DisplayValueSeparator`, but that is also used for other purposes:

- For tag formats that do have official way of having multiple values, for example FLAC with vorbis tags and ID3v2.4, we split the string with our defined `DisplayValueSeparator` which is currently a UTF-8 control string so unlikely to be entered into any tags, to get an array of artists/genres, etc. 

Our current workaround is to check the `MetadataFormats` of the track, and if the format have `ID3v2` as short name but not `ID3v2.4` as full name, we replace the `DisplayValueSeparator` back to `/`, but that is not ideal in my opinion.

**Comment by Zeugma440** [comment]
Alright, thanks for clarifying. 

That's a very specific thing to ask. To be honest, I'd normally say that's a client issue, not a library issue.

However, I understand that Jellyfin recently switched from another library to ATL and built a part of its features around the behaviour of the previous library, which is a delicate situation I'd love to help you solve.

As what you're asking does not add complexity to the library, I'm okay with implementing it.

I'll make it part of next release, as soon as the other recently opened issues are solved.

**Comment by Zeugma440** [comment]
Fix is available in today's v6.05. Please close this issue if it works on your side~

**Comment by gnattu** [comment]
Confirmed working
